lib/inferno/dsl/fhir_evaluator/evaluator_util.rb

```diff
@@ -0,0 +1,81 @@
+module Util
```

Comment:
From discussion - delete this file but preserve Steve's comments in a GitLab issue for when we eventually bring this in

Reply:
Deleted